### PR TITLE
Update create-conditional-access-intune.md

### PR DESCRIPTION
**Pull request by msbemba**
Added permissions required to perform the action

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1870

File: memdocs/intune/protect/create-conditional-access-intune.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ With Intune, enhance Conditional Access in Azure Active Directory by adding mobi
 
 A Conditional Access policy specifies the app or services you want to protect, the conditions under which the apps or services can be accessed, and the users the policy applies to. Although Conditional Access is an Azure AD premium feature, the Conditional Access node you access from *Intune* is the same node as accessed from *Azure AD*.
 
+To Create a device-based Conditional Access policy your account must have one of the following permissions in Azure AD:
+
+Global Administrator	
+Intune Service Administrator	
+Conditional Access Administrator
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/create-conditional-access-intune.md:41]:
```suggestion
    - Global administrator	
    - Intune Service administrator	
    - Conditional Access administrator	
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by Brenduns** [comment]
@yogkumgit @tiburd   Question: This edit ended up introducing a warning for the main branch during daily publishing. It turns out the edits for bullet points that start on line 39 are indented and should not be.  I didn't see a build report when this came in (Still don't see it in this web view) so I didn't notice that.  But when the PR was submitted, that type of build issue should have been flagged. 

Did either of you see a build report showing the indent problem when this submission was first made?  

I'll review future edits more closely if the PR's don't show me a build report! :)

**Comment by tiburd** [comment]
@Brenduns 
Hi, Brent,
I don't think we see build reports for public repo PRs...
Thanks!

@msbemba @JohanFreelancer9 
Please note Brent's comment above regarding not indenting bullets in Markdown.
Thanks!

**Comment by msbemba** [comment]
Thank you @tiburd. I will make a note.

**Pull request merged**